proto/internalapi/sensor/collector.proto

```diff
@@ -21,6 +21,13 @@ enum ExternalIpsEnabled {
   ENABLED = 1;
 }
 
+enum ExternalIpsDirection {
+  UNSPECIFIED = 0;
```

Comment:
I am not sure about this option. The install team has been looking at how defaults should be set. I think there work on that might be relevant here.

Reply:
This is a fair point, I will definitely invite them to review this before considering merging.

Regarding the `UNSPECIFIED` value, I tried to follow the best practice of protobuf. One of the benefits of this approach is that you can change the default behavior in the code, without changing the proto definition.

Reply:
@stackrox/install, could you please take a look and tell me whether this way of managing the default value is acceptable, please ?